### ISSUE TITLE
"Use User" feature lets me encode a video, but when I do it uses the users_id of the original login.

### ISSUE THREAD

**Issue by walt93** [WWBN/AVideo#7274]
**Is your feature request related to a problem? Please describe.**
1. I login with my admin account, users_id=1
2. I go to the Users control panel, and "Use this user" for another user, say it's id=100
3. I go to encode a video from user who's users_id=100
4. That video is encoded and gets users_id=1

**Describe the solution you'd like**
It seems the video should be encoded with users_id=100 in this scenario.

**Comment by DanielnetoDotCom** [comment]
can you please screenshot it?

**Comment by walt93** [comment]
Daniel it's exactly as I stated but let me try again with pictures.

Login with account with users_id == 1:
![Screen Shot 2022-10-17 at 2 43 47 PM](https://user-images.githubusercontent.com/11711050/196289573-10f4ef18-db41-4ff8-bad0-4dedf8e77e15.png)

**Comment by DanielnetoDotCom** [comment]
where does it say id=100??

**Comment by walt93** [comment]
100 was a randomly made up example. See above.

**Comment by DanielnetoDotCom** [comment]
you said this

**Comment by walt93** [comment]
Look at what was emitted to the database. 

I did "Use this user" on "Red Pill News" which has a users_id of 905;

I encoded a video.

That video has a users_id == 1 (the account I logged into before selecting "use this user")

**Comment by DanielnetoDotCom** [comment]
why should be 905?

905 is the videos_id, NOT the users_id

**Comment by walt93** [comment]
Daniel - "SELECT id, title,created,users_id from videos". 

I am selecting the users_id.

It is == 1

It should be == 905 🤦‍♂️

The video's id is 13058. It's under the column "id". The "users_id" column is 1.

**Comment by DanielnetoDotCom** [comment]
now I am even more confused, I just checked again your screenshots and I can confirm 905 is the users_id NOT the videos_id

I think the thing is that the top menu bar is cached, and change the user did not update it.

the only solution now is to clear the cache after changing the user, (only the admin can do that)

in the future, I will need to change the way you access the encoder.

**Comment by walt93** [comment]
Ok, it seems like there is something weird going on with caching and changing users - not going to lie - I don't pretend to understand how caching/clearing cache works with regards to this. But I will try to add some cache clearing steps and see if I can coerce the correct behavior out of it.

**Comment by DanielnetoDotCom** [comment]
that is right, for now, you cannot encode videos for someone else without knowing his password. as I told you before, to let you do it I need to change the way you open the encoder.

**Comment by walt93** [comment]
Ok I just wasn't understanding how this feature is supposed to work. My workflow has me setting up 24 users, encoding a ton of media under their accounts. I thought this would perhaps eliminate some trips to "sudo mysql" to hack my database.

**Comment by DanielnetoDotCom** [comment]
I guess now the only issue is to encode files, you may be able to do everything else.

you do not need to hack the MySQL, admins can change the video owned just editing the video and changing it

**Comment by walt93** [comment]
Daniel I just uploaded 500 files. To edit them manually in the user interface would literally take hours. The app is a bloated memory pig and is extremely cumbersome to do things like this in bulk.

One tiny example. In the /mvideos list simply switching from 10 at a time to 25 at a time, I have to step away from the computer for a minute to let it finish as it takes about a full minute to do that.

![Screen Shot 2022-10-17 at 6 54 05 PM](https://user-images.githubusercontent.com/11711050/196317005-3512d8f8-14e4-4eda-b3da-a2f7f1b42e02.png)

Workflow: scroll down find video, hit edit icon, wait for screen to come up, switch tabs. You're talking thirty seconds *minimum* per video.

Then after you've edited a video and hit save, you're waiting another 20-30 seconds for the screen to refresh. TBH it's painful.

Some workflows are infinitely easier in the terminal.
